repose-aggregator/components/filters/keystone-v2-filter/src/test/scala/org/openrepose/filters/keystonev2/KeystoneV2FilterTest.scala

```diff
@@ -2298,6 +2298,70 @@ with HttpDelegationManager {
     }
   }
 
+  describe("Handling multiple tenant extraction") {
+    def configuration = Marshaller.keystoneV2ConfigFromString(
+      """<?xml version="1.0" encoding="UTF-8"?>
+        |<keystone-v2 xmlns="http://docs.openrepose.org/repose/keystone-v2/v1.0">
+        |    <identity-service
+        |        uri="https://some.identity.com"
+        |        set-groups-in-header="false"
+        |    />
+        |    <tenant-handling>
+        |        <validate-tenant>
+        |            <uri-extraction-regex>/serverOne/(.+)/.*</uri-extraction-regex>
+        |            <uri-extraction-regex>/serverTwo/(.+)/.*</uri-extraction-regex>
+        |            <uri-extraction-regex>/serverToo/(.+)/.*</uri-extraction-regex>
+        |        </validate-tenant>
+        |    </tenant-handling>
+        |</keystone-v2>
+      """.stripMargin)
+
+    val filter = new KeystoneV2Filter(mockConfigurationService, mockAkkaServiceClientFactory, mock[AtomFeedService], mockDatastoreService)
+
+    filter.init(mockFilterConfig)
+    filter.KeystoneV2ConfigListener.configurationUpdated(configuration)
+    filter.SystemModelConfigListener.configurationUpdated(mockSystemModel)
+
+    val testUris = Seq("One", "Two", "Too")
+    testUris foreach { uri =>
```

Comment:
Is there any reason not to inline this variable? Especially since you are describing the values in the function passed to `foreach`.